index.html

```diff
@@ -4,22 +4,27 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Simple hotel website</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css" />
+    <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-    <header>
-        <img src="images/download 1.svg" alt="Hotel logo">
-        <nav>
-            <ul>
-                <li><a href="#our hotel">Our hotel</a></li>
-                <li><a href="#rooms&rates">Rooms & rates</a></li>
-                <li><a href="#facilities">Facilities</a></li>
-                <li><a href="#contact us">Contact us</a></li>
-            </ul>
-            <button type="button"><img src="images/bi_search.svg" alt="loop icon"></button>
-        </nav>
+    <header class="page-header">
+        <div class="container">
+            <img class="header-icon" src="images/download 1.svg" alt="Hotel logo">
```

Comment:
it is better to name assets in one style

common practice :

Name of folders in camelCase 
Name of files in kebab-case (media files - img, svg)

Reply:
and also add a word describing the icon into class name (as you did in the button below "header-loop-icon")

Reply:
extensions of media files:

SVG - icons only
PNG - images without backgrounds, usually a logo, etc.
JPG - content images (downloaded in different extensions 1x - for regular screens, 2x - retina, 3x - even greater pixelation of screens + for each screen in a separate size - mobile up to 768px, tablet up to 1280/1440px mostly, desktops anything wider than tablets )
WEBP is a new optimized format, but not yet supported in all browsers

in this case "download 1" must be as png like a logo